pkg/operator/ceph/object/spec.go

```diff
@@ -103,6 +103,11 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 	if reflect.DeepEqual(rgwDaemonContainer, v1.Container{}) {
 		return v1.PodTemplateSpec{}, errors.New("got empty container for RGW daemon")
 	}
+
+	hostNetwork := c.clusterSpec.Network.IsHost()
+	if c.store.Spec.HostNetwork != nil {
```

Comment:
Later around line 206 the host network is also considered, this will also need to be applied there.

Reply:
done

Reply:
`portString()` in config.go have similar condition

Reply:
> `portString()` in config.go have similar condition
sorry, missed committing this change.